src/Symfony/Component/Messenger/Transport/RedisExt/Connection.php

```diff
@@ -71,6 +71,10 @@ public static function fromDsn(string $dsn, array $redisOptions = [], \Redis $re
 
     public function get(): ?array
     {
+        if (null === $this->blockingTimeout && !$this->count()) {
```

Comment:
Not needed aslong as there is no blockingTimeout xreadgroup is non blocking.

Reply:
well, passing `null` or `0` as `$i_block` when calling `xreadgroup` means "wait infinitely", removing the check makes the test hangs until the socket timeouts

Reply:
Ok did only test it using the redis-cli. Sounds for me this could then be an issue in the phpredis extension.

Reply:
I agree, but we need to workaround it for now